### DISCUSSION TITLE
Multiple Editors with one Toolbar

### DISCUSSION THREAD

**Discussion by stuarteaton** [facebook/lexical#4925] [General]
Is there any sensible way to have multiple editors and one toolbar that is not rendered within the editor. Something that would work for a structure similar to this.

```
<DocumentEditor>
    <Toolbar />
    <Page-editor />
    <Page-editor />
    <Page-editor />
    <Page-editor />
</DocumentEditor>
```

All on one page built from various components. How could I attach the toolbar to the current active Editor? Have played round with context providers etc. but no joy.

**Comment by waqarilyas** [comment]
Any update on this? @stuarteaton were you able to solve this issue?

**Comment by ashiq-prospero** [comment]
I am also waiting for this feature

**Comment by ivailop7** [comment]
Have you tried a single toolbar that dispatches commands based on the activeEditor? I don't see why this wouldn't work.
